apps/server/src/modules/group/controller/mapper/group-response.mapper.ts

```diff
@@ -76,4 +77,11 @@ export class GroupResponseMapper {
 
 		return mapped;
 	}
+
+	static mapToGroupListResponse(groups: GroupDto[]): GroupEntryResponse[] {
+		const mapped: GroupEntryResponse[] = groups.map(
+			(group: GroupDto): GroupEntryResponse => new GroupEntryResponse({ id: group.id, name: group.name })
+		);
+		return mapped;
```

Comment:
Please remember the last code reviews where we have to add a empty line before each return statement